fiware-region-sanity-tests/resources/templates/userdata/test.meta.phonehome.template

```diff
@@ -0,0 +1,18 @@
+#cloud-config
+
+# phone_home: if this dictionary is present, then the phone_home
+# cloud-config module will post specified data back to given url
+#
+runcmd:
+ - curl "http://169.254.169.254/openstack/latest/meta_data.json" > /tmp/metadata.json
```

Comment:
This is a "fake" constant URL defined by OpenStack Metadata Service. It must be exactly that. However, if this may lead to confusion, we could define something like "openstack_metadata_url"